### DISCUSSION TITLE
v2.0.0-beta.4

### DISCUSSION THREAD

**Discussion by sserrata** [PaloAltoNetworks/docusaurus-openapi-docs#631] [Announcements]
## What's Changed
* [Bug] Use SchemaItem to render leaf discriminator properties/nodes by @sserrata in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/573
* [Enhancement] Introduce new theme component names: ApiExplorer, CodeSnippets by @blindaa121 in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/577
* Unify tabs naming by @Dr-Electron in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/612
* [UI] Cleanup schema item and explorer panel caret alignment by @sserrata in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/624
* Fix demo panel for `FormData` request type by @jk8 in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/625
* Prepare release v2.0.0-beta.4 by @sserrata in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/629

## New Contributors
* @Dr-Electron made their first contribution in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/612
* @jk8 made their first contribution in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/625

**Full Changelog**: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/compare/v2.0.0-beta.3...v2.0.0-beta.4

<hr /><em>This discussion was created from the release <a href='https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/releases/tag/v2.0.0-beta.4'>v2.0.0-beta.4</a>.</em>

**Comment by Mikescops** [comment]
@sserrata do you mind tagging a new .5 version with the new changes as it unblocks people using PNPM? 
That would be lovely as we're trying to upgrade our vulnerable dependencies :)

**Reply by sserrata** [reply]
Hi @Mikescops, have you tested with the latest canary beta release?

**Reply by Mikescops** [reply]
you mean the .4 version? if so yes I did, hence why I submitted PRs 👍

**Reply by Mikescops** [reply]
ooooh never mind, I understood, there is another channel ok [0.0.0-beta.672](https://www.npmjs.com/package/docusaurus-plugin-openapi-docs/v/0.0.0-beta.672), I'll give it a try thanks a lot!

**Reply by Mikescops** [reply]
works like a charm, thank you 👍
